app/src/main/java/org/oppia/android/app/story/StoryFragmentPresenter.kt

```diff
@@ -142,6 +146,50 @@ class StoryFragmentPresenter @Inject constructor(
             ).parseOppiaHtml(
               storyItemViewModel.summary, binding.chapterSummary
             )
+
+          val str = if (storyItemViewModel.chapterSummary.chapterPlayState ==
+            ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES
+          ) {
+            String.format(
+              HtmlCompat.toHtml(
+                SpannedString
+                  .valueOf(view.context.getText(R.string.unlock_chapter)),
+                HtmlCompat.TO_HTML_PARAGRAPH_LINES_CONSECUTIVE
+              ),
+              storyItemViewModel.index,
+              storyItemViewModel.missingPrerequisiteChapter.name
+            )
+          } else {
+            String.format(
+              HtmlCompat.toHtml(
+                SpannedString.valueOf(
+                  view.context
+                    .getText(R.string.chapter_name)
+                ),
+                HtmlCompat.TO_HTML_PARAGRAPH_LINES_CONSECUTIVE
+              ),
+              storyItemViewModel.index + 1,
+              storyItemViewModel.name
+            )
+          }
+          binding.storyChapterCard.strokeWidth =
+            when (storyItemViewModel.chapterSummary.chapterPlayState) {
+              ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES ->
+                view.context.resources.getDimension(R.dimen.space_0dp).toInt()
+              else ->
+                view.context.resources.getDimension(R.dimen.story_chapter_stroke_width).toInt()
+            }
+          binding.titleContent =
+            htmlParserFactory.create(
+              resourceBucketName,
+              entityType,
+              storyItemViewModel.storyId,
+              imageCenterAlign = true
+            ).parseOppiaHtml(
+              str,
+              binding.chapterTitle
+            )
+
```

Comment:
Remove this extra line. This is causing the lint test error.
To avoid this I suggest to setup the [ktlint](https://github.com/oppia/oppia-android/wiki#installation) pre-push for the project

Reply:
sure